<launch>


  <node name="statistical_outlier_removal"
        pkg="nodelet" type="nodelet"
        args="standalone pcl/StatisticalOutlierRemoval">
    <remap from="~input" to="model_cloud" />
    <rosparam>
      mean_k: .1
      stddev: 1.0
    </rosparam>
  </node>

</launch>
